src/tools/annotation/bidirectionalTool/moveHandle/getBaseData.test.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-eval */
+import getBaseData from './getBaseData';
```

Comment:
Is `getBaseData` public API?

Reply:
Nope, it's being used only as a helper for moving handles from bidirectional tool lines.